### DISCUSSION TITLE
테스트환경에서 RoomJoined가 호출되지 않습니다.

### DISCUSSION THREAD

**Discussion by YoungBok-Wi** [naverz/zepeto-studio-kor#415] [Issue Report (Legacy)]
프로젝트 제작중에 유니티환경에서는 잘 작동하는데, 테스트환경에서는 제대로 작동되지 않아서
여러가지 테스트를 해봤는데, RoomJoined가 호출되지 않는것같네요.
이건 제페토 버그인가요 제가 뭔가 빼먹은건가요ㅜㅜ

새로 프로젝트를 만들어서 해도 똑같은 문제가 발생합니다.
테스트한 제페토앱 버전은 3.9.8이고, 월드패키지 버전은 1.1.4입니다.

원래 프로젝트는 분량이 꽤 많은 관계로 테스트용 프로젝트 코드를 올립니다.

--------------------------------------------------------------------------------------------------------------------------
TestScript.ts

    import { Text } from 'UnityEngine.UI'
    import { ZepetoScriptBehaviour } from 'ZEPETO.Script'
    import { ZepetoWorldMultiplay } from 'ZEPETO.World';
    
    export default class TestScript extends ZepetoScriptBehaviour {
        public m_LogText:Text;
        public m_Multi:ZepetoWorldMultiplay;
    
        Start() {
            this.m_Multi.RoomCreated += (_room) => {
                this.m_LogText.text += "Created\n";
            };
            this.m_Multi.RoomJoined += (_room) => {
                this.m_LogText.text += "Joined\n";
            };
            this.m_Multi.RoomError += (_err) => {
                this.m_LogText.text += _err + "\n";
            };
        }
    }

index.ts

    import { Sandbox, SandboxOptions, SandboxPlayer } from "ZEPETO.Multiplay";
    
    export default class extends Sandbox {
    
        onCreate(options: SandboxOptions) {
            
        }
    
        onJoin(client: SandboxPlayer) {
    
        }
    
        onLeave(client: SandboxPlayer, consented?: boolean) {
            
        }
    }

실행 결과 (텍스트의 기본값은 Load\n입니다)
![Screenshot_20220222-162449_ZEPETO](https://user-images.githubusercontent.com/8111902/155083252-9043caa8-977b-4c69-a37c-e83a732a267e.jpg)

--------------------------------------------------------------------------------------------------------------------------
그리고 원래 프로젝트에서 index.ts의 onJoin에서 broadcast 함수를 호출했을때는 RoomError가 호출되었습니다.
code : 4216
message : this.broadcast is not a function

**Comment by x4cf** [comment]
1.1.4 버전에서 룸 크리에이트와 룸 조인, 프로드케스트 모두 정상작동합니다 :)
